### ISSUE TITLE
Task info sidepane "Assign to someone" field is buggy

### ISSUE THREAD

**Issue by RobH779** [Charlie9830/pounder#110]
**Describe the bug**
When clicking on "Assign to someone" in the task info sidepane, instead of a text entry field popping up it displays a weird glyph and doesn't allow you to enter a username.

**What were you doing at the time?**
Clicked on "Assign to someone" in task info sidepane to enter a username, instead weird glyph suddenly appeared. Sadly it was not MissingNo. and I was unable to duplicate item 6 in my inventory.

**Expected behavior**
Expected a text box to display to allow me to enter a username to assign the task too.

**Screenshots**
![Screen Shot 2019-05-26 at 9 37 41 pm](https://user-images.githubusercontent.com/41365436/58381374-bc0f0600-7fff-11e9-8c87-5097383906a0.png)
_Figure 1: Weird glyph instead of text box_

**Version Information**
 - OS: macOS 10.14.4
 - Version 2.2.1

**Comment by Charlie9830** [comment]
Do you have other members collaborating within that project? 
It seems that it is allowing you to open the Assign menu as you are in a shared project, but isn't rendering any options, either because you have no other Members in the project, or for some reason it has failed to load them.

**Comment by RobH779** [comment]
Yeah there are definitely other members sharing that project. Also worth mentioning that -a rob and -a tim both don't work either.
